examples/gauge-metrics/gauge_metrics.bal

```diff
@@ -0,0 +1,101 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/io;
+import ballerina/observe;
+
+@http:ServiceConfig { basePath: "/online-store-service" }
+service<http:Service> OnlineStoreService bind { port: 9090 } {
+
+    //Create a gauge as a global varaible in the service with optional field description,
+    //and default statistics configurations = { timeWindow: 600000, buckets: 5,
+    // percentiles: [0.33, 0.5, 0.66, 0.99] }.
+    observe:Gauge globalGauge = new("global_gauge", desc = "Global gauge defined");
+
+    @http:ResourceConfig {
+        path: "/make-order"
+    }
+    makeOrder(endpoint caller, http:Request req) {
+        io:println("------------------------------------------");
+        //Incrementing the global gauge defined by 15.0.
+        globalGauge.increment(amount = 15.0);
+        //Log the current state of global gauge.
+        printGauge(globalGauge);
+
+
+        //Create a gauge with simply a name, and default statistics configurations.
+        observe:Gauge localGauge = new("local_operations");
+        //Increment the local gauge by default value 1.0.
+        localGauge.increment();
+        //Increment the value of the gauge by 20.
+        localGauge.increment(amount = 20.0);
+        //Decrement the local gauge by default value 1.0.
+        localGauge.decrement();
+        //Decrement the value of the gauge by 20.
+        localGauge.decrement(amount = 10.0);
+        //Log the current state of local gauge.
+        printGauge(localGauge);
+
+
+        //Create a gauge with optional fields description and tags defined.
+        observe:Gauge registeredGaugeWithTags = new("registered_gauge_with_tags",
+            desc = "RegisteredGauge", tags = { property: "gaugeProperty", gaugeType: "RegisterType" });
+
+        //Register the gauge instance, therefore it is stored in the global registry and can be reported to the
```

Comment:
Apply changes as indicated in previous comments.

Reply:
As mentioned in the above comment, I don't think it's a good idea to change it as suggested.